app/src/common/constants/permissions.js

```diff
@@ -91,7 +91,7 @@ export const PERMISSIONS_MAP = {
     [ACTIONS.EDIT_OWN_ACCOUNT]: ALL,
     [ACTIONS.DELETE_LAUNCH]: OWNER,
     [ACTIONS.EDIT_LAUNCH]: OWNER,
-    [ACTIONS.FORCE_FINISH_LAUNCH]: OWNER,
```

Comment:
Force finish action should be available for not own launches only with `rerun` label